distributed/minGPT-ddp/mingpt/slurm/sbatch_run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Do the template, README and sbatch script need to be different for this PR and PR # 2?

Reply:
- sbatch script is only different in the path of the script to run.
- yaml.template and the cluster setup instructions are identical and independent of the script we're running